### DISCUSSION TITLE
Plotly Spectral Band Mapping

### DISCUSSION THREAD

**Discussion by kevinhicks7** [sertit/eoreader#43] [Q&A] [answered]
Does anyone know where I can find the source code that made this plot https://eoreader.readthedocs.io/en/latest/_static/optical_band_mapping.html? I feel like I've looked everywhere but I may have missed something obvious.

**Comment by remi-braun** [comment, marked as answer]
Hello,

That code is not in EOReader but sleeps peacefully on my computer 😛 
Do you need it for some reason?
It is quite dirty I'm afraid 😅
